sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapRfRExamples.cs

```diff
@@ -0,0 +1,195 @@
+                                TestDataUtilities.exampleDiscountFactors2, "1D"));
+                        break;
+                    case "SONIA":
+                        upsertComplexMarketDataRequest.Add("discount_curve_GBP",
+                            TestDataUtilities.BuildRateCurveRequest(TestDataUtilities.EffectiveAt, "GBP", "OIS",
+                                TestDataUtilities.exampleDiscountFactors1));
+                        upsertComplexMarketDataRequest.Add("projection_curve",
+                            TestDataUtilities.BuildRateCurveRequest(TestDataUtilities.EffectiveAt, "GBP", "SONIA",
+                                TestDataUtilities.exampleDiscountFactors2, "1D"));
+                        break;
+                    case "TONA":
+                        upsertComplexMarketDataRequest.Add("discount_curve_JPY",
+                            TestDataUtilities.BuildRateCurveRequest(TestDataUtilities.EffectiveAt, "JPY", "OIS",
+                                TestDataUtilities.exampleDiscountFactors1));
+                        upsertComplexMarketDataRequest.Add("projection_curve",
+                            TestDataUtilities.BuildRateCurveRequest(TestDataUtilities.EffectiveAt, "JPY", "TONA",
+                                TestDataUtilities.exampleDiscountFactors2, "1D"));
+                        break;
+                    case "SARON":
+                        upsertComplexMarketDataRequest.Add("discount_curve_CHF",
+                            TestDataUtilities.BuildRateCurveRequest(TestDataUtilities.EffectiveAt, "CHF", "OIS",
+                                TestDataUtilities.exampleDiscountFactors1));
+                        upsertComplexMarketDataRequest.Add("projection_curve",
+                            TestDataUtilities.BuildRateCurveRequest(TestDataUtilities.EffectiveAt, "CHF", "SARON",
+                                TestDataUtilities.exampleDiscountFactors2, "1D"));
+                        break;
+                }
+                upsertComplexMarketDataResponse =
+                    _complexMarketDataApi.UpsertComplexMarketData(scope, upsertComplexMarketDataRequest);
+                ValidateComplexMarketDataUpsert(upsertComplexMarketDataResponse,
+                    upsertComplexMarketDataRequest.Count);
+            }
+        }
+
+        /// <inheritdoc />
+        protected override void GetAndValidatePortfolioCashFlows(
+            LusidInstrument instrument,
+            string scope, string portfolioCode,
+            string recipeCode,
+            string instrumentID)
+        {
+            var swap = (InterestRateSwap) instrument;
+            var cashflows = _transactionPortfoliosApi.GetPortfolioCashFlows(
+                scope: scope,
+                code: portfolioCode,
+                effectiveAt: TestDataUtilities.EffectiveAt.AddYears(3),
+                windowStart: swap.StartDate.AddDays(-3),
+                windowEnd: swap.MaturityDate.AddDays(3),
```

Comment:
Maybe add a few more days, or just make it a month. You don't want to accidently miss any cashflows because of roll conventions. It would be a pretty hard bug to spot

Reply:
This is an example of how to get cashflows - the window is arbitrary and might only be used to extract specific cashflows - doesn't necessarily need to get all cashflows.